src/table/utils/selections-utils.js

```diff
@@ -129,10 +133,24 @@ const selectCell = (state, payload) => {
   return { ...state, rows: selectedRows, colIdx: -1 };
 };
 
+const selectMultiValues = (state) => {
+  const { api, rows, colIdx, isSelectMultiValues } = state;
+
+  isSelectMultiValues &&
+    api.select({
+      method: 'selectHyperCubeCells',
+      params: ['/qHyperCubeDef', Object.values(rows), [colIdx]],
+    });
+
+  return { ...state, rows, colIdx };
```

Comment:
isSelectMultiValues should be set to false here

Reply:
and you are not changing anything else, so spread that:
```suggestion
  return { ...state, isSelectMultiValues: false };
```

Reply:
you should also add a default value in TableContext